### DISCUSSION TITLE
N64 Transfer Pak Support?

### DISCUSSION THREAD

**Discussion by Sowden** [darthcloud/BlueRetro#739] [General]
Hey there.  I don't think anyone has mentioned this, but I'm wondering if Transfer Pak support could ever be inlmplamated.  I mean, you would have to apply a ROM with your saves.  And you would have to personally dump your cart if you wanted your personal saves.  It would be a bit of a one, two step process, but I wonder if it could ever be done.

**Comment by Sowden** [comment]
*looks around* Anybody?

**Comment by Sowden** [comment]
Nobody?  Is this not a desired thing for Pokemon Snap or Mario Golf 64?

**Reply by supavgx** [reply]
you should check out usb64 its a dead project as developer went silent but it implements it

**Comment by darthcloud** [comment]
It's possible but flash space remaining is very limited on ESP32 so a whole GBC ROM wouldn't fit.

I'm always open for PR so I could happen if someone else does it.

Other than that I'm very busy and that's quite a niche feature. So unlikely to happen.

**Reply by Sowden** [reply]
Gotcha.  Yeah it is rather niche, and I'm sure it's not asked for a lot.  Thanks for getting back
